README.md

```diff
@@ -1,40 +1,37 @@
-# Codewind Che Plugin
-The Codewind plugin for Eclipse Che!
+# Codewind Che Plug-in
+Use the Eclipse Codewind sidecar plug-in for Eclipse Che to enable Theia to communicate with the Codewind server container.
 
 [![Eclipse License](https://img.shields.io/badge/license-Eclipse-brightgreen.svg)](https://github.ibm.com/dev-ex/tempest/blob/master/LICENSE)
```

Comment:
Should this license be removed? It is pointing to a URL that isn't accessible externally.

Reply:
Oh, yes, we can remove it.